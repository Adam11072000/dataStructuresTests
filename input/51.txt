init 7025
addImage 710
deleteImage 710
mergeSuperPixels 2509 -6 6662
addImage 9384
resetLabelScore 279 -2 1466
deleteImage -10
resetLabelScore 7800 4000 -3
addImage -10
addImage 1352
quit