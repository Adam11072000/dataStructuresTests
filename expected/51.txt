init done.
addImage: SUCCESS
deleteImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
deleteImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
addImage: SUCCESS
quit done.
